.gitignore

```diff
@@ -108,3 +108,6 @@ scan-journal.log
 
 # mvn
 target/
+
+# CDK load artifacts
+airbyte-cdk-load/
```

Comment:
nit: we can probably make this more precise:
```
airbyte-cdk/bulk/core/load/airbyte-cdk-load/
airbyte-integrations/connectors/*airbyte-cdk-load/
```

(i.e. exclude specifically the directories where we expect this to show up, rather than everywhere in the repo)

Reply:
ah! Good call. looks like `airbyte-integrations/connectors/*airbyte-cdk-load/` would actually need to be `airbyte-integrations/connectors/**/airbyte-cdk-load/`. I'll update

Reply:
addressed in [1c331bd](https://github.com/airbytehq/airbyte/pull/48620/commits/1c331bd6d9d04e1da077990613b41ef864078485)

Reply:
derp, this is what I get for writing code directly in a pr comment :P 

:shipit: